common/validators.js

```diff
@@ -97,5 +97,42 @@
                 // });
             }
         };
+    })
+
+    .directive('timestamp', function() {
+        return {
+            restrict: 'A',
```

Comment:
@jrchudy Hmm I'm not sure I understand your comment here. What do you mean by "define this as an element directive as  a child element to input?" Is this what you mean?
```
<timestamp validate-values="someValues">
    <input date .....>
</timestamp>
```
I'm also not convinced that this should be an element directive over an attribute directive.